Makefile.work

```diff
@@ -14,6 +14,14 @@
 #  * USERNAME: Desired username -- default at rules/config
 #  * PASSWORD: Desired password -- default at rules/config
 #  * KEEP_SLAVE_ON: Keeps slave container up after building-process concludes.
+#  *                Please note that with current stretch build structure,
+#  *                user of KEEP_SLAVE_ON feature will have to be concious
+#  *                about which docker to stay on.
+#  *                - If user desire to stay on stretch docker, please issue
```

Comment:
desire -> desires

Also, maybe it should read, "If user desires to keep the Stretch Docker running, please issue..."

Reply:
Thanks, changed now.